packages/esm-hiv-app/src/index.ts

```diff
@@ -47,6 +52,7 @@ function setupOpenMRS() {
   clearCareAndTreatmentSidenavRegistry();
   clearHivPreventionSidenavRegistry();
 
+  addToBaseFormsRegistry(hivForms);
```

Comment:
@pirupius note what happens here, hiv provides it's custom forms to the framework. Something similar needs to happen for covid